evmbin/src/display/json.rs

```diff
@@ -195,7 +194,24 @@ impl trace::VMTracer for Informant {
 			// print last line with final state:
 			self.gas_cost = 0.into();
 			let gas_used = self.gas_used;
-			self.trace_executed(gas_used, &[], &[]);
+			let subdepth = self.subdepth;
+
+			Self::with_informant_in_depth(&mut self, subdepth, |informant: &mut Informant| {
+				let info = ::evm::Instruction::from_u8(informant.instruction).map(|i| i.info());
+
+				let trace = json!({
```

Comment:
Any way we could get rid of this duplication? It's not strongly typed and having two places where this is generated is imho error prone in the long term.